build-tools/src/scripts/Dockerfile

```diff
@@ -8,7 +8,11 @@ ARG GID
 # so we need to declare it explicitly
 # this must be an ARG so it doesn't carry through post-build phase
 ARG all_proxy
+# hadolint ignore=DL3018
 RUN apk add --no-cache openssh-client git gcc linux-headers libc-dev util-linux libpcap-dev bash vim make protobuf protobuf-dev sudo tar curl graphviz ttf-freefont
+# we need updated libraries, here we use the same version as for eve/alpine
+# hadolint ignore=DL3018
+RUN apk --no-cache --repository https://dl-cdn.alpinelinux.org/alpine/v3.13/main add -U --upgrade zfs-dev zfs-libs
```

Comment:
Here we build an image that we use to build other parts (and eve/alpine as well). I am not happy with looped dependencies. Actually the reason to have additions here is because of tests (make test) require them.